### ISSUE TITLE
Ability to specify path to WebView2

### ISSUE THREAD

**Issue by NanoNik** [wailsapp/wails#1446]
### Is your feature request related to a problem? Please describe.

I'm building an application using wails v2, what I want to do is to download webview2 in a separate bootstrapper that will download all dependencies of the application (including webview2, if not installed), and then use downloaded (not installed in the system) version.

I think it's not very good for user experience to pop up messages and force them to download a webview or silently download it without any visual feedback.

Also, this could be helpful when debugging issues with a specific version of webview2.

### Describe the solution you'd like

Basically just add a new field in struct `windows.Options` and pass it to `createCoreWebView2EnvironmentWithOptions`

```
options := &windows.Options {
    WebviewBrowserPath: "webview_downloaded/",
}
```

**Comment by leaanthony** [comment]
I see a benefit in this and am happy to accept a PR for it. It sounds like you already know what needs doing 👍

**Comment by D06F6E67** [comment]
@leaanthony  Does WebViewBrowserPath only support absolute paths? My configuration of relative paths doesn't seem to work, but absolute paths do. How do I specify a relative path?

Specifying `./build/windows/WebView2Runtime` in `main.go` doesn't work.
Specifying `D:/myapp/build/windows/WebView2Runtime` works.
File directory structure:
myapp/
├─ main.go
├─ build/
│ ├─ windows/
│ │ └─ WebView2Runtime/
│ │ ├─ msedgewebview2.exe
│ │ ├─ ...
